app/assets/stylesheets/styles/game.scss

```diff
@@ -1,11 +1,13 @@
-/*// * {margin: 0; padding: 0; }
-// table { border-collapse: collapse; border-spacing: 0; }
-//
-// #chessboard { padding: 0px; margin: 0 auto; border: 2px solid #181818; }
-// #chessboard tr td { width: 60px; height: 60px; }
-// #chessboard tr:nth-child(2n) td:nth-child(2n+1) { background: #9f9f9f; }
-// #chessboard tr:nth-child(2n+1) td:nth-child(2n) { background: #9f9f9f; }*/
-
+#gamesheader {
+  color: #9bb0bb;
```

Comment:
Color literals like `#9bb0bb` should only be used in variable declarations; they should be referred to via variable everywhere else.